README.md

```diff
@@ -292,7 +292,8 @@ A curated list of awesome inclusive events/projects/initiatives for women in the
     The Womoz Blog. Program updates and news from the women and Mozilla project.
 
   - [InspirAda na Computação](https://inspiradanacomputacao.com/) - *Loc:* <img src="https://upload.wikimedia.org/wikipedia/en/0/05/Flag_of_Brazil.svg" alt="Brazil" width="30"> - *Lang: PT-BR only*
-  - The InspirAda na Computação is a blog made by Lidiane Monteiro, a PyLady. The blog is focused on the space of the women in the computation.
+   
+    The InspirAda na Computação is a blog made by Lidiane Monteiro, a PyLady. The blog is focused on the space of the women in the computation.
```

Comment:
From original: Compartilhamos conteúdo como forma de estimular mulheres para área de computação, promover o acesso ao conhecimento técnico e trazer como resultado a igualdade de oportunidades

How about:
The blog aims to share content in order to inspire women to become interested in the tech area. It promotes the access to the tech knowledge and brings, as a result, the equality of opportunities.

Reply:
Yes, that looks much better actually.